dds_cli/utils.py

```diff
@@ -190,6 +190,19 @@ def perform_request(
     # Check if response is ok.
     if not response.ok:
         message = error_message
+
+        if response.status_code == http.HTTPStatus.BAD_REQUEST:
+            if DDSEndpoint.CREATE_PROJ in endpoint:
+                message += f": {__project_creation_error(response_json)}"
+
+            raise dds_cli.exceptions.DDSCLIException(message=message)
+
+        if response.status_code == http.HTTPStatus.FORBIDDEN:
+            if DDSEndpoint.CREATE_PROJ in endpoint:
+                message += ": You do not have the required permissions to create a project."
+
+            raise dds_cli.exceptions.DDSCLIException(message=message)
```

Comment:
I have created this now, will continue filling it in: https://github.com/ScilifelabDataCentre/dds_web/pull/1164 
This documents which status codes are returned by the API endpoints and when. 

Just adding this here because this also means that this if statement means that there will be an incorrect message when there are not enough unit admins. 

I think we should remove this check for now and add a better handling of FORBIDDEN at a later time.